utils/zerovec/src/ule/plain.rs

```diff
@@ -98,6 +109,16 @@ macro_rules! impl_byte_slice_size {
                     })
                 }
             }
+
+            /// This function can be used for constructing ZeroVecs in a const context, avoiding
+            /// parsing checks.
+            ///
+            /// See [`ZeroSlice`] for an example.
+            pub const fn from_ule_slice_const(slice: &[RawBytesULE<$size>]) -> &Self {
```

Comment:
nit: i don't think this should have `const` in the name; it's generally useful

Reply:
There is already such a (non const) function on ZeroSlice.

Naming suggestions?

Reply:
ahhh. hm. `from_raw_bytes_slice`

Reply:
I'm also fine with it as is if there's a conflicting method